app/src/main/java/com/example/keybindhelper/RecyclerViewAdapters/ProjectAdapter.java

```diff
@@ -115,7 +114,7 @@ public void onBindViewHolder(@NonNull ProjectViewHolder holder, int position) {
                     p.updateLastAccessed();
                     DatabaseManager.db.update(p);
                     notifyItemChanged(position);
-                    showSnackBarMessage("Renamed to \'"+text+"\'!");
+                    showSnackBarMessage("Renamed to "+text+"!");
```

Comment:
"/" un-needed

Reply:
It looks like the group wants the single quote, which was also removed.
I'd consider changing this to "Renamed to '$text' "
The \ may still be needed in that case.